draft-ietf-tls-tls13.md

```diff
@@ -1817,6 +1817,17 @@ the 0-RTT data without detection. If the client's ephemeral share
 is compromised and client authentication is used, then the attacker
 can impersonate the client on subsequent connections.
 
+It is up to the implementation and application layers above TLS to mitigate
+the impact of these weaknesses.  For example, to support strong client
+authentication and forward secrecy over 0-RTT connections, it is possible to
+issue a new ticket on the first 1-RTT connection, and on every resumed session
+after that.  Each co-located group of servers could ensure that tickets they
+issue are used only once by maintaining a local cache of ticket decryption
+keys.  An accurate clock on the client could ensure freshness of client
+signatures.  In addition it is a good idea to make all request idempotent,
```

Comment:
I think that you will find that clock sync is basically unattainable in practice.  However, I don't think that it's necessary.  Since the server generates the challenge (even if it's on a previous connection), it is entirely in control of freshness.  All it has to do is tune the window over which it will accept 0-RTT.  This makes the client clock irrelevant here.